app/models/organization/OrganizationService.scala

```diff
@@ -77,15 +80,18 @@ class OrganizationService @Inject()(organizationDAO: OrganizationDAO,
       existingOrganization <- organizationDAO.findOneByName(organizationName)(GlobalAccessContext).futureBox
       _ <- bool2Fox(existingOrganization.isEmpty) ?~> "organization.name.alreadyInUse"
       initialPricingPlan = if (conf.Features.isDemoInstance) PricingPlan.Basic else PricingPlan.Custom
+      organizationRootFolder = Folder(ObjectId.generate, organizationName)
       organization = Organization(ObjectId.generate,
                                   organizationName,
                                   "",
                                   "",
                                   organizationDisplayName,
-                                  initialPricingPlan)
+                                  initialPricingPlan,
+                                  organizationRootFolder._id)
       organizationTeam = Team(ObjectId.generate, organization._id, "Default", isOrganizationTeam = true)
       _ <- organizationDAO.insertOne(organization)
       _ <- teamDAO.insertOne(organizationTeam)
+      _ <- folderDAO.insertAsRoot(organizationRootFolder)
```

Comment:
double-check that this works, what about the foreign key constraint? changing the order might help